src/main/java/me/totalfreedom/totalfreedommod/command/Command_linkdiscord.java

```diff
@@ -0,0 +1,50 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import java.util.Random;
+import me.totalfreedom.totalfreedommod.admin.Admin;
+import me.totalfreedom.totalfreedommod.discord.Discord;
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.SUPER_ADMIN, source = SourceType.ONLY_IN_GAME)
+@CommandParameters(description = "Link your Discord account to your Minecraft account", usage = "/<command>")
+public class Command_linkdiscord extends FreedomCommand
+{
+
+    @Override
+    public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (!plugin.dc.enabled)
+        {
+            msg("The Discord verification system is currently disabled.", ChatColor.RED);
+            return true;
+        }
+
+        Admin admin = plugin.al.getAdmin(playerSender);
+        if (admin.getDiscordID() != null)
+        {
+            msg("Your Minecraft account is already linked to a Discord account.", ChatColor.RED);
```

Comment:
What happens in the event you want to over-write your discord ID? I'd be inclined to allow it? OR at the very least we tell people on the command to un-link first?

Reply:
Unlink and re-link

Reply:
@ZeroEpoch1969 Have players been confused around this before or just tend to get that they need to un-link and re-link?